src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -51,7 +50,7 @@ public RecalcEngine(PowerFxConfig powerFxConfig)
         }
 
         // Set of default functions supported by the interpreter. 
-        private static readonly ReadOnlySymbolTable _interpreterSupportedFunctions = ReadOnlySymbolTable.NewDefault(Library.FunctionList);
+        private static readonly ReadOnlySymbolTable _interpreterSupportedFunctions = ReadOnlySymbolTable.NewDefault(Library.FunctionList.ToTexlFunctionSet());

```

Comment:
>NewDefault [](http://example.com/codeflow?start=105&length=10)

Add overload:
ReadOnlySymbolTable.NewDefault<TValue>(IReadOnyDictionary<TexlFunction, TValue> keys) 

And then you can completely remove Library.FunctionList!